models/classes/runner/RunnerParamParserTrait.php

```diff
@@ -202,4 +202,63 @@ protected function getItemRef($itemIdentifier)
         $mapService = $this->getServiceLocator()->get(QtiRunnerMap::SERVICE_ID);
         return $mapService->getItemHref($serviceContext, $itemIdentifier);
     }
+
+    protected function setNavigationContextToCommand(object $command): void
+    {
+        $command->setNavigationContext(
```

Comment:
Would it be possible to rely on interfaces somehow, as `object` in type hint looks too generic to safely use `::setNavigationContext` in its context?

Reply:
Makes sense, I was really considering that but I thought that would be fine since it's not a public method.